spec/integration/volumes_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'spec_helper'
+require 'time'
+require 'fileutils'
+
+describe "chef-provisioning-docker" do
+  extend DockerSupport
+  include DockerConfig
+
+  when_the_chef_12_server "exists", organization: "foo", server_scope: :context, port: 8900..9000 do
+    with_docker "volumes supported" do
+
+      # owing to how RSpec works, things defined by let() are not accessible in the recipes we define inside
+      # expect_converge{}.
+      ubuntu_options = {
+          :base_image => {
+            :name => 'ubuntu',
+            :repository => 'ubuntu',
+            :tag => '14.04'
+          }
+        }
+
+      let(:timestamp) { Time.now.to_i }
+      docker_driver = driver
+
+      context "machine_image resource" do
+
+        let(:spec_image_tag) { "docker_image_spec_#{timestamp}" }
+
+        after(:each) {
+          image = docker_driver.find_image("chef:#{spec_image_tag}")
```

Comment:
This produces `ArgumentError: wrong number of arguments (1 for 2)`: it's `find_image(repository, image_tag)`.